packages/sources/svelte/src/components/structure/VtmnCard/test/VtmnCard.spec.js

```diff
@@ -56,6 +56,13 @@ describe('VtmnCard', () => {
       expect(getSlotImg(container)).toBeVisible();
       expect(getImg(container)).toBeVisible();
     });
+    test('Should set the title as h1', () => {
+      const { getByRole } = render(VtmnCard, {
+        title: 'unit test',
+        headingLevel: 1,
+      });
+      expect(getByRole('heading', { level: 1 })).toBeVisible();
+    });
   });
```

Comment:
Can you please add a test by default the title are into a span ?
Thanks :)

Reply:
All changes mentioned have been made. Thanks @Tlahey ;)